src/localization/fr/cheatsheet.json

```diff
@@ -0,0 +1,70 @@
+{
+  "cheatsheet.section.title": "Aide-mémoire sur les Regex",
+  "cheatsheet.section.description": "Explorez tous les modèles et symboles Regex avec l'aide-mémoire Regex.",
+
+  "cheatsheet.anchors": "Ancres",
+  "cheatsheet.caret.title": "Début de la chaîne ou de la ligne",
+  "cheatsheet.caret.description": "Correspond au début de la chaîne ou de la ligne.",
+  "cheatsheet.dollar.title": "Fin de la chaîne ou de la ligne",
+  "cheatsheet.dollar.description": "Correspond à la fin de la chaîne ou de la ligne.",
+  "cheatsheet.wordBoundary.title": "Limite de mot",
+  "cheatsheet.wordBoundary.description": "Correspond au caractère du mot ou à la position à la fin d'un mot.",
+  "cheatsheet.notWordBoundary.title": "N'est pas Limite de mot",
```

Comment:
If it's not intended I would remove the capital letter " L " on line 12 since it is in the middle of the sentence.

Line 12 would become : 
"cheatsheet.notWordBoundary.title": "N'est pas limite de mot",

If you take into account the comment about the plural and add the " s " to " mot " :
"cheatsheet.notWordBoundary.title": "N'est pas limite de mots",

Reply:
In the first line I wrote :
"If it's not intended I would remove the capital letter " L " on line 12 since it is in the middle of the sentence".
To be slightly clearer do not remove it but instead replace it with the same non-capitalized letter.

Reply:
In the first line I wrote :
"If it's not intended I would remove the capital letter " L " on line 12 since it is in the middle of the sentence".
To be slightly clearer do not remove it but instead replace it with the same non-capitalized letter.

Reply:
Hi @ImMorph 🖐
Thanks for taking time to review my code with a great explanation.. 
Merci !